pkg/auth/requests/authenticate.go

```diff
@@ -158,15 +156,48 @@ func (a *tokenAuthenticator) Authenticate(req *http.Request) (*AuthenticatorResp
 	authResp.User = token.UserID
 	authResp.UserPrincipal = token.UserPrincipal.Name
 	authResp.Groups = groups
-	logrus.Debugf("Auth filter calling provider to get extra info %v", token.AuthProvider)
+	authResp.Extras = getUserExtraInfo(token, u, attribs)
+	logrus.Infof("Extras returned %v", authResp.Extras)
```

Comment:
This sounds more like Debug level message rather than Info. Same for the other `Infof` below.